### PR TITLE
Add optional data version control (dvc.org) support

### PR DESCRIPTION
**Pull request by shcheklein**
First step to fix https://github.com/drivendata/cookiecutter-data-science/issues/158

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ The directory structure of your new project looks like this:
 │   ├── interim        <- Intermediate data that has been transformed.
 │   ├── processed      <- The final, canonical data sets for modeling.
 │   └── raw            <- The original, immutable data dump.
+├── data.dvc           <- A data version conrol file (optional); see dvc.org for details
```
Comment (dmpetrov) [inline README.md]: typo - "conrol"

File: .gitignore [hunk 2]
```diff
@@ -6,4 +6,4 @@ docs/site/
 # test cache
 .cache/*
 tests/__pycache__/*
-*.pytest_cache/
\ No newline at end of file
+*.pytest_cache/
```
Comment (dmpetrov) [inline .gitignore]: I'd better keep new line symbol

**Comment by shcheklein** [comment]
@dmpetrov thanks! published the fix

**Comment by stevesimmons** [comment]
Hi @dmpetrov ... Will this be merged anytime soon? Thanks

**Comment by dmpetrov** [comment]
Hi @stevesimmons

Unfortunately, I'm not a maintainer of this project. I just made a code review. It might be better to ask @isms

But as you can see from the discussion #158 - it is very unlikely that the change will be merged. We need to create a separate cookiecutter-data-science with DVC support.

**Comment by ciberger** [comment]
> Hi @stevesimmons
> 
> Unfortunately, I'm not a maintainer of this project. I just made a code review. It might be better to ask @isms
> 
> But as you can see from the discussion #158 - it is very unlikely that the change will be merged. We need to create a separate cookiecutter-data-science with DVC support.

Hey @dmpetrov, do you know if someone already created an alternative project with DVC support?

**Comment by pjbull** [comment]
Cleaning up PRs that aren't planned to merge

**Review by fugufisch — COMMENTED** [review]
👍🏻

**Pull request closed without merging**